pkg/core/disk.go

```diff
@@ -58,11 +60,32 @@ func (d *DiskOption) Validate() error {
 			return fmt.Errorf("unknown units of size : %s, DiskOption : %v", d.Size, d)
 		}
 	}
+
 	if d.Action == DiskFillAction {
 		if d.FillByFallocate && byteSize == 0 {
 			return fmt.Errorf("fallocate not suppurt 0 size or 0 percent data, "+
 				"if you want allocate a 0 size file please set fallocate=false, DiskOption : %v", d)
 		}
+
+		_, err := os.Stat(d.Path)
```

Comment:
do we need to add the same judgment for the write payload?

Reply:
added